### ISSUE TITLE
Add a minion that powers up by consuming body parts and growing them on itself

### ISSUE THREAD

**Issue by SoftMonster** [miki151/keeperrl#1688]
Add a minion that powers up by consuming body parts and growing them on itself. This would be really neat.

**Comment by miki151** [comment]
Yes, that's what I'm planning for a special minion of the necromancer keeper. It will use the automatons logic, just with gathered body parts. Unless you mean something different.

**Comment by miki151** [comment]
Btw, you could already mod this in using the automaton system.
